oshmem/mca/spml/ucx/spml_ucx.c

```diff
@@ -323,6 +321,8 @@ int mca_spml_ucx_add_procs(ompi_proc_t** procs, size_t nprocs)
     free(wk_roffs);
 
     SPML_UCX_VERBOSE(50, "*** ADDED PROCS ***");
+
+    opal_common_ucx_mca_proc_added();
```

Comment:
why is this needed?

Reply:
there is detection of missing event handler. original call was from incorrect function